docs/src/i18n.md

```diff
@@ -1,29 +1,47 @@
-# Internationalisation and localization
+# Internationalization and localization
 
 ## Date and time formats
 
-We use some [config files](https://github.com/QwantResearch/erdapfel/tree/master/language/date) to format the dates and opening hours.
+We use [config files](https://github.com/QwantResearch/erdapfel/tree/master/language/date) inspired from [the i18n config of jquery-ui](https://github.com/jquery/jquery-ui/tree/master/ui/i18n) to format the dates and hours.
 
-They are inspired from [the i18n config of jquery-ui project](https://github.com/jquery/jquery-ui/tree/master/ui/i18n).
```

Comment:
This section is obsolete and should be removed.  
We are now using the standard [`Intl.DateTimeFormat`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/DateTimeFormat) to format datetimes in browser, and `language/date` folder no longer exists